tide_event.install

```diff
@@ -11,6 +11,7 @@ use Drupal\user\Entity\Role;
 use Drupal\user\Entity\User;
 use Drupal\workflows\Entity\Workflow;
 use Drupal\Component\Utility\Random;
+use Drupal\Core\Session\AccountInterface;
```

Comment:
This line is already on line 8. https://github.com/dpc-sdp/tide_event/pull/41/files#diff-10e5f9f2647683c6fda5e1c06888efbfL8

I don't think that's why the build is failing.